### PR TITLE
Detecting license type

### LINKED ISSUE #6: Detecting License
Would it be possible to also detect license, like The MIT License (MIT), Apache License, etc?

### PR DESCRIPTION
**Pull request by igorkulman**
Fixes #6

### CODE REVIEW

File: fetch_licenses.swift [hunk 1]
```diff
@@ -62,6 +62,40 @@ struct CartfileEntry: CustomStringConvertible {
...
+        
+        print("Fetching license name for \(name) ...")
+        
+        var request = URLRequest(url: URL(string: "https://api.github.com/repos/\(self.name)")!)
+        request.addValue("application/vnd.github.drax-preview+json", forHTTPHeaderField: "Accept")
+        let task = URLSession.shared.dataTask(with: request, completionHandler: { (data, response, error) -> Void in
+            
+            if let response = response as? HTTPURLResponse {
+                if response.statusCode == 404 {
+                    return
```
Comment (pietbrauer) [inline fetch_licenses.swift]: This would lead to the semaphore waiting forever, am I right?

**Comment by pietbrauer** [comment]
As I see we are now fetching only the name from GitHub put still relying on the License or License.txt to be there, right? Can we fetch the License content and the name in one go?

**Comment by igorkulman** [comment]
Yes, no changes to the way the actual license content is fetched. But you are right, I can fetch the actual license content while at it.

**Comment by pietbrauer** [comment]
Thanks!

**Comment by igorkulman** [comment]
Changed the code, now it also fetches the license content from the Github API.

**Comment by pietbrauer** [comment]
Thanks a lot! Works like a charm

**Pull request merged**